src/AutoUI/Actions/Update.tsx

```diff
@@ -87,38 +142,60 @@ export const Update = <T extends AutoUIBaseResource<T>>({
 			</Box>
 		);
 	}
+	const groupedActions = React.useMemo(
+		() => groupBy(updateActions, (action) => action.section),
+		[updateActions],
+	);
+	const actionHandlers = React.useMemo(
```

Comment:
These hook now need to go above the early return `if (!updateActions || updateActions.length < 1) { return null; }` that we have earlier in this file.

Reply:
ufff you are right, nice catch 🙏